src/pages/authentication/Login.jsx

```diff
@@ -77,10 +99,13 @@ export default function Login() {
             </section>
             <input
               type="submit"
-              className="btn btn-solid-primary margin"
+              className="btn btn-solid-primary auth-btn margin margiin-l-3-5"
               value="Login"
             />
           </form>
+          <button className="btn btn-solid-primary auth-btn margin" onClick={()=>TestLogin()}>
```

Comment:
always name the methods/function in camelCase

Reply:
ok, Thank you for review